Lombiq.Hosting.Azure.ApplicationInsights/Services/TelemetryFilter.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Microsoft.ApplicationInsights.Channel;
+using Microsoft.ApplicationInsights.DataContracts;
+using Microsoft.ApplicationInsights.Extensibility;
+using System.Collections.Generic;
+
+namespace Lombiq.Hosting.Azure.ApplicationInsights.Services;
+
+public class TelemetryFilter : ITelemetryProcessor
+{
+    private readonly List<string> _errors = new()
+    {
+        "Azure.RequestFailedException: The specified container already exists.",
+        "The specified blob does not exist.",
+        "Microsoft.Data.SqlClient.SqlException (0x80131904): There is already an object named",
+    };
+    private readonly ITelemetryProcessor _next;
+
+    public TelemetryFilter(ITelemetryProcessor next) =>
+        _next = next;
```

Comment:
```suggestion
    public TelemetryFilter(ITelemetryProcessor next) => _next = next;
```

Reply:
I've asked you this multiple times recently. Please, commit these code suggestions directly, so the reviewer doesn't have to eyeball such trivial things.

Reply:
Sorry, I don't see these comments that you resolve right away, because if you do multiple comments in the same PR I won't read it in my email. And when I come here I won't read resolved conversations.

Reply:
OK